### ISSUE TITLE
[Windows] Microsoft Defender SmartScreen Blocks Masscode

### ISSUE THREAD

**Issue by GaryFurash** [antonreshetov/massCode#135]
## Describe the bug

Downloading Masscode is blocked by Microsoft Defender

## To Reproduce

Windows 10, Download

**Example:**
massCodeSetup1.3. exe was blocked

**Comment by antonreshetov** [comment]
Duplicate of #38
